### PR TITLE
Setting up index Page

### PR DESCRIPTION
**Pull request by SnakyMz**
Things I have done:

- [ ]  Added HTML file.
- [ ]  Added CSS file.
- [ ]  Added test file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,36 +60,25 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Hello Microverse! <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```
Comment (usorfaitheloho) [inline README.md]: You did well implementing the project requirement and adding the README file usimg the required template, kindly remove all descriptions from your README file as they were added to give you instructions on what to do and not as part of your file. The descriptions in your README usually starts with `>`, please kindly remove all of them 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse!</title>
+</head>
+
+<body>
+    <h1>Hello Microverse!</h1>
```
Comment (usorfaitheloho) [inline index.html]: According to the project requirement, you are to add a class to your `header` and then style the class in your css file, please add a class in the `h1` tag and then go ahead with the styling in the css file 😄 

image

File: README.md [hunk 3]
```diff
@@ -225,9 +144,7 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- No Plans
```
Comment (usorfaitheloho) [inline README.md]: [ ]  Kindly I recommend you to update your README file to add  1-3 potential features to be added 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @SnakyMz  ,
> 
> Great job for implementing some of the Project Requirement, but there are some changes that need to be implemented in order to proceed to the next project 👍
> 
> ## Highlights 👏🏻
> ✔️ No linters error ✔️ You used correct Git Hub-Flow. ✔️ Good README 
> 
> almost 

> ## Required Changes ♻️
> _Check the comments below the review._
> 
> ## Optional suggestions

- [x] kindly change the `master` branch to `main` branch as the `master` naming system is now outdated 😄
- [x] Please use descriptive commit messages written in the imperative tense and capitalized.  [read here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) 

> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -5,7 +5,8 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse!</title>
+    <link rel="stylesheet" href="./style.css">
+    <title class="heading">Hello Microverse!</title>
```
Comment (usorfaitheloho) [inline index.html]: You did well implementing the previous changes , please use the `h1` tag and not `title` like this 

> 
       <h1 class="heading">Hello Microverse </h1>

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @SnakyMz  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (SnakyMz) [reply index.html]: Ok just saw my silly mistake. Thanks a lot.

**Review by usorfaitheloho — APPROVED** [review]
Hi @SnakyMz  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**